man/org.freedesktop.systemd1.xml

```diff
@@ -1492,6 +1502,11 @@ node /org/freedesktop/systemd1 {
       boot loader or initrd implementation. In these cases the respective pairs of timestamps are both 0,
       indicating that no data is available.</para>
 
+      <para><varname>UnitsLoadTimestamp</varname> encode <constant>CLOCK_REALTIME</constant>
+      and <constant>CLOCK_MONOTONIC</constant> microseconds timestamps (as described above). The timestamps
+      are taken every time when the manager loads unit files. It could be used to notify users that
```

Comment:
"loads unit files" → "starts loading unit files" maybe? It's important to mention that this the beginning, so that it is possible to compare this timestamp with a timestamp of a file on disk to unambiguously know that it if the timestamp on disk is earlier, the file was loaded in that version.

"notify users that daemon-reload is needed" — hmm, that doesn't seem right, this property being changed means that it's **not** necessary to do a daemon-reload. Maybe just drop that last sentence.

Reply:
> "notify users that daemon-reload is needed" — hmm, that doesn't seem right, this property being changed means that it's **not** necessary to do a daemon-reload. Maybe just drop that last sentence.

That's a good point, thanks.